test/integration_tests/test_pytorch.py

```diff
@@ -42,11 +46,14 @@ def setUpClass(cls):
         ua_detrac = f"{EVA_ROOT_DIR}/data/ua_detrac/ua_detrac.mp4"
         mnist = f"{EVA_ROOT_DIR}/data/mnist/mnist.mp4"
         actions = f"{EVA_ROOT_DIR}/data/actions/actions.mp4"
+        asl_actions = f"{EVA_ROOT_DIR}/data/actions/computer_asl.avi"
```

Comment:
Is this file uploaded? Is it redundant with "computer_asl.mp4"?

Reply:
i have updated the tests to use computer_asl.mp4. the file is also uploaded.